docs/content/get-started/set-up.md

```diff
@@ -186,22 +199,20 @@ From the root of the `gdk-for-unity-fps-starter-project` repository:
 
 <%(#Expandable title="Manually clone the GDK for Unity")%>
 
-1. Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
-
+1\.  Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
```

Comment:
Only saw it now, but there is a line in this doc saying:
`Then navigate to the root folder of the FPS starter project and run the following command: git checkout 0.1.3`

They don't need to do this anymore. They can use the master branch directly

Reply:
Removed this like